packages/core-plugin-api/api-report.md

```diff
@@ -505,6 +513,11 @@ export type IdentityApi = {
 // @public
 export const identityApiRef: ApiRef<IdentityApi>;
 
+// @public
+export type LazyLoadedPackageJson = () => Promise<{
+  default: Record<string, unknown>;
+}>;
```

Comment:
Probably best avoid this alias for now, while shipping things as experimental. i.e. inline instead

Reply:
Removed export, and named it experimental

Reply:
It's used in two places (which isn't much, but more than 1), so inlining feels so-so